x/rollappparams/types/errors.go

```diff
@@ -0,0 +1,9 @@
+package types
+
+import (
+	errorsmod "cosmossdk.io/errors"
+)
+
+var (
+	ErrDANotSupported = errorsmod.Register(ModuleName, 1, "da type not supported")
+)
```

Comment:
no need for this you can return a wrapped gerrc.ErrInvalidArgument inline

Reply:
removed and replaced by error inline